src/iframe-helper.js

```diff
@@ -586,3 +586,20 @@ export function canInspectWindow(win) {
     return false;
   }
 }
+
+/** @const {string} */
+export const EMBED_PROP = '__AMP_EMBED__';
```

Comment:
Minor: Rename to `FIE_EMBED_PROP` now that this is visible.

Reply:
done